measurement/plugins/netflix_fast/measurements.py

```diff
@@ -105,7 +105,9 @@ def measure(self):
 
         results.append(self._get_fast_result())
         for thread_result in self.thread_results:
-            results.append(self._get_url_result(thread_result))
+            results = results + self._get_url_result(thread_result)
+            print(len(results), "\n")
+        print(results)
```

Comment:
@jamesaj23 you're gonna nix these `print` statements, right? ;-)

Reply:
What print statements 😅